skin/assets/styles/utils/_shared-variables.scss

```diff
@@ -7,6 +7,10 @@ $base-content-padding: 15px;
 $base-content-full-width: 1200px;
 $base-content-width: $base-content-full-width + ($base-content-padding*2);
 
+// Grid
+$base-gutter-size: $base-content-padding;
+$base-col-number: 12;
```

Comment:
All looks good tho maybe we could move these 2 variables away from `_shared-variables.scss`? If we're moving towards moving mixins away from boilerplate and into a library we might want to contain these variables somewhere else. @iruzevic

Reply:
Yes, but until we split this to mixins I'm ok for them to stay in shared_variables.